apps/src/assetManagement/ImagePicker.jsx

```diff
@@ -0,0 +1,71 @@
+var AssetManager = require('./AssetManager.jsx');
+var IconLibrary = require('./IconLibrary.jsx');
+
+/**
+ * A component for managing hosted assets.
+ */
+module.exports = React.createClass({
+  propTypes: {
+    assetChosen: React.PropTypes.func,
+    typeFilter: React.PropTypes.string,
+    channelId: React.PropTypes.string.isRequired,
+    uploadsEnabled: React.PropTypes.bool.isRequired
+  },
+
+  getInitialState: function () {
+    return {mode: 'files'};
+  },
+
+  setIconMode: function () {
+    this.setState({mode: 'icons'});
+  },
+
+  setFileMode: function () {
+    this.setState({mode: 'files'});
+  },
+
+  render: function () {
+
```

Comment:
I've seen this style of having an empty line at the beginning of a function in the PRs of a couple folks now. Is there an argument in favor of it? I've kind of been discouraging it, but if it's something that multiple other folks like, I can stop doing so.